src/v2/view-builder/views/IdentifierView.js

```diff
@@ -90,21 +90,21 @@ const Body = BaseForm.extend({
     }
   },
 
+  handleClearFormError() {
```

Comment:
instead of overriding `handleClearFormError` add `this.$el.find('.o-form-error-container').empty();` to the `handleClearFormError` for `BaseForm` before `if` statement